### ISSUE TITLE
Attempting to drag a top site deletes it

### ISSUE THREAD

**Issue by bbondy** [brave/browser-ios#234]
If you try to drag a top site on the new tab page, even if you are dragging not close the x and out of the button onto a blank area, it will delete the tile.

**Comment by garvankeeley** [comment]
They aren't currently draggable, but I can see the sense in this as you were trying to re-order them

**Comment by bbondy** [comment]
ya I don't think this should be registered as a click to close if someone thinks they can drag and can't.  We should at least not delete the tile :)

**Comment by garvankeeley** [comment]
True, because once deleted, it doesn't come back until the prefs are cleared, that domain is blacklisted from showing on the top sites
